content/SCALE/SCALEUIReference/Storage/Disks/DisksScreens.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "Disks Screens"
+description: "This article provides information on the settings found on and functions of the Disks Screens."
+weight: 30
+aliases:
+ - /docs/scale/scaleuireference/storage/disksscale/
```

Comment:
We don't need the `/docs` portion as that is baked into the base URL of the website (truenas.com/docs).
```suggestion
 - /scale/scaleuireference/storage/disksscale/
```